### DISCUSSION TITLE
synchronize only downloads one file

### DISCUSSION THREAD

**Discussion by giladgressel** [abraunegg/onedrive#1420] [Q&A] [answered]
Hi,

I have setup onedrive on a server (ubuntu 20.04) and have setup a sync_list with a single folder
```
# only one folder in the sync_list
EUS_ML
``` 
I ran re-sync after
When I run `onedrive --synchronize --download-only --dry-run` it shows me that it will download all the files in the folder. However when I run the command without `--dry-run` it downloads one file and then stops.

I've tried with and without `--download-only` and I've also tried specifying the folder with single directory
nothing seems to work

It's an educational onedrive account. I own the folder, but have shared it with others.
Any ideas on how to get it to sync all the files?

one more detail -- I'm running it in a `tmux` session and detaching. I'm wondering if the output is getting weirdly formatted and I'm thinking it's stopped when it hasn't?? Going to check carefully now.

thanks ahead of time!

Here is a screenshot of it ending.
![image](https://user-images.githubusercontent.com/6275366/116427780-f58ec280-a861-11eb-9bfa-1a28b8fb4e52.png)
I tried in tmux and out of tmux. both times it looks like it's going for the next file -- but just stops.

**Comment by abraunegg** [comment, marked as answer]
@giladgressel 
> I'm running it in a tmux session and detaching. 

Don't do this as I do not advise this.

> However when I run the command without `--dry-run` it downloads one file and then stops.

Please follow the correct support process for what looks like a bug. Open a new issue using the Issue Template and fill in all the details.

Also, as you are running Ubuntu 20.04, you most likely will be either running an old version or if you are using the PPA, still an old version. Both are unsupported as they are old and contain issues. You most likely will have to uninstall the package and build + install the client from source.

**Reply by giladgressel** [reply]
Yes I am running from PPA.
I will uninstall and build from source and see if that fixes it. 
if it doesn't I will follow the support process. Thank you very much for your prompt reply and support!

**Reply by giladgressel** [reply]
Just a quick update.
I uninstalled (with purge) and rebuilt from source (great instructions thank you very much).

Seems to be working now. I tested with some random text files.
I will have to test again to make sure it wasn't the video files (.asf) or something strange like that, but seems to have been solved!
thanks a lot

**Reply by abraunegg** [reply]
@giladgressel 
Glad this is resolved. Additionally, due to using the PPA, you would have also been unwittingly impacted by this issue: https://github.com/abraunegg/onedrive/issues/1274

This could have also been a contributing factor.
